README.md

```diff
@@ -95,6 +95,13 @@ wireguard_postup:
 wireguard_postdown:
   - ...
 wireguard_save_config: "true"
+wireguard_unmanaged_peers:
+  client.example.com:
+    public_key: 5zsSBeZZ8P9pQaaJvY9RbELQulcwC5VBXaZ93egzOlI=
```

Comment:
I'm not very familiare with the pre-shared key config option, is that a secret? In this case maybe the README should promote good practices using `ansible-vault`

Reply:
Do you have a change for me to include? Otherwise I'd leave psk out of the example.

It is secret but on its own does no good so I suppose it depends on your threat model whether you'd use Vault for that, and only for that?

Reply:
You can read more about pre-shared keys here: https://wiki.archlinux.org/index.php/WireGuard#Key_generation (2nd half). In general I'd leave `ansible-vault` out. It's simply out of scope. Maybe mentioning `ansible-vault` is ok but that's it. If people want to use it they need to read the manual of `ansible-vault` anyways if they're not already familiar with it.